src/include/daos/object.h

```diff
@@ -443,6 +443,11 @@ int daos_obj_generate_oid_by_rf(daos_handle_t poh, uint64_t rf_factor,
 				daos_oclass_id_t cid, daos_oclass_hints_t hints,
 				uint32_t args);
 
+int daos_obj_generate_oid_by_rf2(daos_handle_t poh, uint64_t rf_factor,
```

Comment:
this is not a public API, so you can just replace the old one with this new one.
no need to add API versioning here.